src/lib/dom.generated.d.ts

```diff
@@ -2265,12 +2307,14 @@ declare var CDATASection: {
     new(): CDATASection;
 };
 
+/** The CSS interface holds useful CSS-related methods. No object with this interface are implemented: it contains only static methods and therefore is a utilitarian interface. */
 interface CSS {
     escape(value: string): string;
     supports(property: string, value?: string): boolean;
 }
 declare var CSS: CSS;
 
+/** An object implementing the CSSConditionRule interface represents a single condition CSS at-rule, which consists of a condition and a statement block. It is a child of CSSGroupingRule. */
```

Comment:
Here a bit more complex: `An object implementing the CSSConditionRule interface`